jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DispatcherTest.java

```diff
@@ -772,7 +772,7 @@ public void testDispatchMapping404() throws Exception
             \r
             """);
         response = HttpTester.parseResponse(rawResponse);
-        assertThat(response.getContent(), containsString("matchValue=DispatchServlet, pattern=/DispatchServlet, servletName=DispatchServlet, mappingMatch=EXACT"));
+        assertThat(response.getContent(), containsString("matchValue=TestServlet, pattern=/TestServlet, servletName=TestServlet, mappingMatch=EXACT"));
```

Comment:
This change was me just reverting the assertion change from commit https://github.com/eclipse/jetty.project/commit/9efce1580422fe35a978070e58ec4f0c53241afc#diff-2db903d841617ed7161f2eecd4945d01ddcddbe5e1e4ee0e937349f7c4d9d1cbL706